site/docs-md/guides/firebase-push-notifications.md

```diff
@@ -0,0 +1,368 @@
+# Using Push Notifications with Firebase in an Ionic Angular 4 App
+
```

Comment:
Suggestion about evergreen content: It's been challenging for me (and likely others) to keep version numbers etc up to date over time. Even though it's a tiny update, the mental aspect and actual effort takes more time than you'd think.  so.... :)

Consider mentioning "Ionic 4" specifically a bit less. here could be:
Web Framework: Ionic 4 + Angular

as i've got in my guide. then "Ionic App" over "Ionic 4" because future versions (5 and beyond) aren't going to change much.